katacomb/requirements.txt

```diff
@@ -43,9 +42,11 @@ requests
 scipy
 toolz
 tornado
-webencodings == 0.5.1
+typing-extensions
 urllib3
+
 katdal @ git+https://github.com/ska-sa/katdal
 katpoint @ git+https://github.com/ska-sa/katpoint
-katsdpservices @ git+https://github.com/ska-sa/katsdpservices
+katsdpsigproc @ git+https://github.com/ska-sa/katsdpsigproc
 katsdptelstate @ git+https://github.com/ska-sa/katsdptelstate
+katsdpservices @ git+https://github.com/ska-sa/katsdpservices
```

Comment:
Why did this move below telstate? They were alphabetical before.

Reply:
Whoops! I fixed this is the latest commit.

Assuming this passes the tests you can take a look now.